tests/restart_on_config_change_test.yml

```diff
@@ -2,13 +2,13 @@
 - hosts: all
   tasks:
     - name: get docker process PID
-      command: "pgrep -f '/usr/bin/dockerd --raw-logs'"
+      command: "pgrep -f '/usr/bin/docker daemon --raw-logs'"
       register: r_docker_pid
 
 - hosts: all
   vars:
     # same version as already installed
-    docker_version: 1.12.1
+    docker_version: 1.11.2
```

Comment:
This got changed to something lower?

Is 1.11.2 basically our new formerly up to date most recent docker version we're using, but the world has raced past us and it's no longer the latest presumably?